js_task2/src/scss/app-style.scss

```diff
@@ -0,0 +1,216 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+  a {
+    @include app-font;
+    text-decoration: none;
+    color: $font-secondary;
+  }
+
+  .account-actions {
+    display: flex;
+    div {
+      display: flex;
+      @include app-font;
+      color: $font-secondary;
+
+      &::after {
+        content: '';
+        display: block;
+        margin: 0 8px;
+        width: 1.5px;
+        height: 100%;
+        background-color: $font-secondary;
+        opacity: 0.1;
+      }
+    }
+  }
+}
+
+.page-name {
+  padding: 12px;
+  background-color: $background-primary;
+  color: $white;
+  @include app-font;
+}
+
+.filter-section {
+  border-bottom: 1.5px solid rgba(128, 128, 128, 0.171);
+  padding: 12px;
+  .filter-form {
+    justify-content: space-between;
+    flex-wrap: wrap;
+
+    fieldset {
+      display: flex;
+      align-items: center;
+      flex-direction: row;
+      width: 100%;
+      border: none;
+
+      legend {
+        @include app-font;
+        color: $font-secondary;
+      }
+
+      .filter-inputs {
+        display: flex;
+        justify-content: flex-start;
+        flex-direction: column;
+        width: 100%;
+        @include desktop-size {
+          flex-direction: row;
+        }
+        input {
+          padding: 10px 8px;
+          border: 2px solid rgba(214, 214, 214, 0.719);
+          border-radius: 2px;
+          outline: none;
+          margin-bottom: 12px;
+          width: 100%;
```

Comment:
please keep box model order, width, and margin should be above

Reply:
done